REST/PowerShell/Reports/DeploymentsByMachineReport.ps1

```diff
@@ -0,0 +1,263 @@
+$ErrorActionPreference = "Stop";
+
+# Fix ANSI Color on PWSH Core issues when displaying objects
+if ($PSEdition -eq "Core") {
+    $PSStyle.OutputRendering = "PlainText"
+}
+
+$stopwatch = [system.diagnostics.stopwatch]::StartNew()
+
+# Define working variables
+$octopusURL = "https://your.octopus.app"
```

Comment:
I had to add a check to ensure these variables were updated to the GetUsageCounts.ps1 script (https://github.com/OctopusDeploy/OctopusDeploy-Api/blob/master/REST/PowerShell/Administration/GetUsageCounts.ps1#L9) because users didn't know and weren't told that they needed to replace these values. When the users reported the error, AEs didn't know what was going on and passed it through to the SEs. A similar check here would be useful to close this feedback loop.